desktop-exporter/server.go

```diff
@@ -13,51 +14,92 @@ type Server struct {
 	traceStore *TraceStore
 }
 
-func getTracesHandler(traceStore *TraceStore) func(http.ResponseWriter, *http.Request) {
+func newTraceSummary(traceID string, spans []SpanData) (*TraceSummary, error) {
```

Comment:
You don't need the `traceID` technically, you could pick that out of any of the `SpanData`s since they should all have the same `traceID`